src/main.ts

```diff
@@ -64,8 +65,20 @@ withTelemetry(
       }
 
       Sentry.setTag('sourcemaps', sourcemaps.length > 0);
+      Sentry.setTag('inject', inject);
```

Comment:
this is a tag for which step they reach?

I am just now realizing that the plugins set a unique transaction for debugid uploads, not just a tag, but the ask is really just simple: Can i measure that a run included sourcemaps upload? And can i messure which of those were with debugIds injected.